### DISCUSSION TITLE
LCL fillter in GFM model

### DISCUSSION THREAD

**Discussion by Leonawyciii** [CURENT/andes#573] [Q&A] [answered]
Hi Developers,

Is there an existing LCL filter module in ANDES? If not, when using the REGF models, is it necessary to manually add the LCL filter during modelling?

Additionally, if the LCL filter is not added, will it significantly affect the simulation results based on your experience? I'm particularly concerned about how it might impact the accuracy of voltage and current waveforms and reduced-order model.

Thanks

**Comment by cuihantao** [comment, marked as answer]
The REGF models are phasor-based stability models. The filters are of a smaller timescale and should not impact the phasor-based average models.

**Reply by Leonawyciii** [reply]
Thanks!
